pydarn/plotting/fan.py

```diff
@@ -153,23 +154,22 @@ def plot_fan(cls, dmap_data: List[dict], ax=None,
         if isinstance(scan_index, dt.datetime):
             # loop through dmap_data records, dump a datetime
             # list where scans start
-            scan_datetimes = np.array([dt.datetime(*[record[time_component]
-                                                     for time_component in
-                                                     ['time.yr', 'time.mo',
-                                                      'time.dy', 'time.hr',
-                                                      'time.mt', 'time.sc']])
-                                       for record in dmap_data
-                                       if record['scan'] == 1])
-            # find corresponding scan_index
-            matching_scan_index = np.argwhere(scan_datetimes ==
-                                              scan_index)[..., 0] + 1
+            scan_time = scan_index
+            scan_index = 1
```

Comment:
```suggestion
            scan_index = 0
```
The time is shifted by one minute atm